backend/de.metas.business/src/main/java/de/metas/order/model/interceptor/C_OrderLine.java

```diff
@@ -481,8 +485,14 @@ private void updateIsOnConsignmentFromLines(final List<OrderId> orderIds)
 		}
 	}
 
+	@ModelChange(timings = { ModelValidator.TYPE_BEFORE_NEW })
+	public void copyDimensionFromHeader(final org.compiere.model.I_C_OrderLine orderLine)
+	{
+		final I_C_Order order = orderBL.getById(OrderId.ofRepoId(orderLine.getC_Order_ID()));
+		orderLine.setM_SectionCode_ID(order.getM_SectionCode_ID());
 
-
-
+		final Dimension orderDimension = dimensionService.getFromRecord(order);
+		dimensionService.updateRecordUserElements(orderLine, orderDimension);
```

Comment:
pls shortly docu why u used updateRecordUserElemens + setM_SectionCode_ID instead of simply using updateRecord.